readthedocs/settings/base.py

```diff
@@ -433,6 +433,7 @@ def TEMPLATES(self):
     # instance to avoid file permissions issues.
     # https://docs.docker.com/engine/reference/run/#user
     RTD_DOCKER_USER = 'docs:docs'
+    RTD_BUILD_SUPER_USER = 'root:root'
```

Comment:
```suggestion
    RTD_DOCKER_SUPER_USER = 'root:root'
```

Keep consistency with the above setting.